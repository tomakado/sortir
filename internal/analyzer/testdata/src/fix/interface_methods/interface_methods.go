package interface_methods

type UnsortedInterface interface {
	ZMethod()
	AMethod() // want "interface methods are not sorted"
	BMethod()
}

type AnotherInterface interface {
	Write([]byte) (int, error)
	Close() error // want "interface methods are not sorted"
	Read([]byte) (int, error)
}