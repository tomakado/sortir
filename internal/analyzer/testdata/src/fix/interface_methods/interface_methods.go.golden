package interface_methods

type UnsortedInterface interface {
	AMethod() // want "interface methods are not sorted"
	BMethod()
	ZMethod()
}

type AnotherInterface interface {
	Close() error // want "interface methods are not sorted"
	Read([]byte) (int, error)
	Write([]byte) (int, error)
}